src/components/overflow-menu/_overflow-menu.scss

```diff
@@ -329,8 +329,16 @@
     max-width: 11.25rem;
     transition: outline $transition--base, background-color $transition--base, color $transition--base;
 
+    svg {
+      fill: $icon-02;
+    }
+
     &:hover {
       color: $text-01;
+
+      svg {
+        fill: $text-01;
```

Comment:
Should this be `$icon-01` ?

Reply:
yes should be resolved now @alisonjoseph